src/config.ts

```diff
@@ -233,36 +241,45 @@ export class Config {
     fs.removeSync(this.path(p));
   }
 
-  async askWriteProjectFile(
+  async confirmWriteProjectFile(
     path: string,
     content: any,
-    force?: boolean,
     confirmByDefault?: boolean,
-  ): Promise<void> {
+  ): Promise<boolean> {
     const writeTo = this.path(path);
-    let next = true;
-    if (typeof content !== "string") {
-      content = JSON.stringify(content, null, 2) + "\n";
+    if (!fsutils.fileExistsSync(writeTo)) {
+      return true;
     }
-    let existingContent: string | undefined;
-    if (fsutils.fileExistsSync(writeTo)) {
-      existingContent = fsutils.readFile(writeTo);
+    const existingContent = fsutils.readFile(writeTo);
+    const newContent = stringifyContent(content);
+    if (existingContent === newContent) {
+      utils.logBullet(clc.bold(path) + " is unchanged");
+      return false;
     }
-    if (existingContent && existingContent !== content && !force) {
-      next = await confirm({
-        message: "File " + clc.underline(path) + " already exists. Overwrite?",
-        default: !!confirmByDefault,
-      });
+    const shouldWrite = await confirm({
+      message: "File " + clc.underline(path) + " already exists. Overwrite?",
+      default: !!confirmByDefault,
+    });
+    if (shouldWrite) {
```

Comment:
This seems reversed - shouldn't it be !shouldWrite

Reply:
Thanks a bunch!

Missed the case when file changed in manual testing.